glom/core.py

```diff
@@ -636,10 +636,18 @@ def from_text(cls, text):
         Path('a', 'b', 'c')
 
         """
+        def create():
+            segs = text.split('.')
+            segs = [
+                _T_STAR if seg == '*' else
+                _T_STARSTAR if seg == '**' else seg
+                for seg in segs]
+            return cls(*segs)
```

Comment:
Let's get this printing a warning here instead, as we're breaking the default API a bit. We can maybe allow enabling the new behavior with a global flag. Also if we can have the warning link to docs that'd be great.